amplify/ts-code/src/db/ScheduleTable.ts

```diff
@@ -130,6 +173,32 @@ export class ScheduleTable {
             }
         }
         return this.client.get(params)
-            .then((output: DocumentClient.GetItemOutput) => output.Item as ScheduleSchema)
+            .then((output: DocumentClient.GetItemOutput) => {
+                if (output) { 
+                    return output.Item as ScheduleSchema
+                } else {
+                    throw Error(`Reservation not found: ${id}`)
```

Comment:
Actually, this error should be thrown up in the `api` layer. Just return null or undefined if unable to find it. For a generic get, you don't want to throw exceptions if you're unable to find something. (Think about if you're unable to find something in a generic Java array. A exception doesn't get thrown. Instead, it's just returns null)